pylint/testutils/pyreverse.py

```diff
@@ -110,6 +110,35 @@ def get_functional_test_files(
     return test_files
 
 
+def get_functional_test_modules(
```

Comment:
I would prefer `packages` over `modules` here and in the remaining function.  
IMO that fits better to the common understanding what the [difference between modules and packages](https://stackoverflow.com/a/7948672/4030694) is.

Reply:
Alright, I have renamed all occurences of "module" to "package"